modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -66,7 +71,20 @@ def allow_processing?
       Flipper.enabled? :claims_hourly_slack_error_report_enabled
     end
 
-    def get_unique_errors
+    def get_filtered_unique_errors
+      unique_errors = unique_errors_by_transaction_id # unique by transaction_id
```

Comment:
Variable name looks good! I suppose we donʼt need the comment now?

Reply:
oh shoot I thought I had removed it, yep cleaning that up now